cloud/docker.go

```diff
@@ -249,3 +249,42 @@ func (m *dockerManager) GetContainers(ctx context.Context, h *host.Host) ([]stri
 
 	return ids, nil
 }
+
+func (m *dockerManager) RemoveLeastRecentlyUsedImageID(ctx context.Context, h *host.Host) error {
+	images, err := m.client.ListImages(ctx, h)
+	if err != nil {
+		return errors.Wrap(err, "Error listing images")
+	}
+	// get earliest created image ID
+	earliest := time.Now()
```

Comment:
You aren't removing the least recently used image - you're removing the oldest image. I don't think that's a problem necessarily, but you should either implement LRU or rename this to oldest